tools/scripts/create-json-for-manifest.ps1

```diff
@@ -137,7 +137,7 @@ function CreateJsonForManifest([string]$msiBasePath, [string]$octokitRelativePat
     $info.installer_url = "https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v$paddedVersion/$MsiName"
     $info.release_notes_url = "https://www.github.com/Microsoft/accessibility-insights-windows/releases/tag/v$paddedVersion"
     $info.current_version = $paddedVersion
-    $info.production_minimum_version = Get-MinimumProductionVersion $paddedVersion $octokitRelativePath $IsMandatoryProdUpdate
```

Comment:
In Powershell, script parameters are global. This code was using `$IsMandatoryProdUpdate` (upper case 'I') instead of the passed-in value of `$isMandatoryProdUpdate` (lower case 'i'). Basically, we were getting the correct value but had poor encapsulation.